src/sentry/models/outbox.py

```diff
@@ -58,6 +59,8 @@ class OutboxCategory(IntEnum):
     PROJECT_UPDATE = 8
     API_APPLICATION_UPDATE = 9
     SENTRY_APP_INSTALLATION_UPDATE = 10
+    TEAM_UPDATE = 11
+    TEAM_NO_CASCADE = 12
```

Comment:
Do we need both of these? Having the cascade treated like other updates is preferable as it prevents us from processing a cascade *before* any update operations that may be waiting for delivery.